boot/bootdata/.gitattributes

```diff
@@ -0,0 +1,2 @@
+# This one will be distributed in the ISO.
+readme.txt text eol=crlf
```

Comment:
Also, in boot/bootdata/bootcdregtest/ we have the testboot.bat.sample that may need CRLF as a reference.

Reply:
Fixed in 34875c2